feature-utils/poly-look/test/react-components/overlays/sideSheet.test.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { SideSheet } from "../../../src/react-components";
+
+/**
+ * @jest-environment jsdom
+ */
+describe("SideSheet", () => {
+  test("Creates an empty container with close buttons", () => {
```

Comment:
Describe usually includes "it" tests, not "test"; please check the rest of the tests around for consistency.
To be clear, the two options are usually
* Standalone `test` sentences.
* Groups of `it` sentences surrounded by `describe` to create sets of subtests.

In this case, I would say `describe` does not add any functionality, so please consider just eliminating it.

Reply:
Well, `describe`/`it` would be RSpec style indeed, but I dunno about `test` functions generally being at the top level with Jest, in their docs they do it this way too: https://jestjs.io/docs/api#describename-fn

Given that we don't have much consistency with how the tests work right now, I guess it has to be up to individual taste... For me it works either way - if the `describe` makes the test output easier to parse I'd keep it, if we already see e.g. the file name the tests are in, I'd think it's a bit redundant.

Reply:
This bit I've added right after doing the same for the buttons. `SideSwiper` has a `describe` but that's because I've tried (and failed) to write some more advanced tests. I would still like to keep it.